app/controllers/container-type.js

```diff
@@ -31,26 +23,52 @@ export default Controller.extend({
       name: 'Name'
     }];
 
-    this.sortProperties = ['name'];
+    // By default, sort alphabetically
+    this.sorts = [
+      {
+        valuePath: 'name',
+        isAscending: true
+      }
+    ]
   },
 
   /**
    * Inspect an instance in the object inspector.
    * Called whenever an item in the list is clicked.
    *
    * @method inspectInstance
-   * @param {Object} obj
+   * @param {Object} instance
    */
-  inspectInstance: action(function(obj) {
-    if (!get(obj, 'inspectable')) {
+  inspectInstance: action(function(instance) {
+    if (!get(instance, 'inspectable')) {
```

Comment:
```suggestion
    if (!instance.inspectable) {
```

Reply:
Let's also flip this logic around and do `if (instance.inspectable) { ... }` instead of the early return